### ISSUE TITLE
Hello, ive used this program for over 1 and a half years, and today i downloaded the updated version and i got a Trojan...

### ISSUE THREAD

**Issue by moto006** [mrpond/BlockTheSpot#571]
- [x] You are using the Spotify `last tested version` mentioned in the [README](https://github.com/mrpond/BlockTheSpot/blob/master/README.md).
 - [x] You are using the latest [BlockTheSpot](https://www.github.com/mrpond/BlockTheSpot/releases) from github.
 - [x] You uninstalled Spotify and just ran the patch
 - [x] You read through the [known issues](https://github.com/mrpond/BlockTheSpot/blob/master/README.md#known-issues) section in the README.
 - [x] You read through the [additional notes](https://github.com/mrpond/BlockTheSpot/blob/master/README.md#additional-notes) section in the README.
 - [x] You enabled logging by editing config.ini `set log = 1`.
 - [x] Make sure whether an issue related yours is already open or not.
 - [x] Make sure you have completed everything in the above checklist.

## Note: 
- Antivirus detections are false positive. Please only open an issue if you find any vulnerability in the code. False reports will be closed without notice.
- If we find that you have opened an issue without following the troubleshooting above, the issue will be closed without notice.

**Comment by PhoenixR49** [comment]
I've got the same problem...

**Comment by lewill03** [comment]
This is honestly really weird, I mean devs say this is a false positive (ofc), but then what tf did they change in the code that make dpapi.dll now triggers 29 (!!) detections on virustotal instead of only 2 for the July version??

**Comment by xinthose** [comment]
I don't trust it anymore.

**Comment by ThePastaGuy** [comment]
Yeah it infected my Windows defender, so now I'm resetting my pc. These people have obviously been hacked or are just plain scum and they/the hackers are using this program to spread malware. DON'T DOWNLOAD ANY UPDATE AND DELETE OFF YOUR DEVICE IMMEDIATELY

**Comment by cuicui-V5** [comment]
same problem

**Comment by theQwackdoc** [comment]
So now even Block the Spot is gone? Any way we can fix this issue?

**Comment by ThePastaGuy** [comment]
Honestly this is one of the worst infections my pc has ever had. And I've had accounts stolen and shit. Not one of those viruses went after windows defender and won tho. Really scary. Also ez fix for spotify on pc, get an extension that blocks the ads in browser. It's not gonna eat up more resources unless you use chrome

**Comment by Izkrul** [comment]
> Honestly this is one of the worst infections my pc has ever had. And I've had accounts stolen and shit. Not one of those viruses went after windows defender and won tho. Really scary. Also ez fix for spotify on pc, get an extension that blocks the ads in browser. It's not gonna eat up more resources unless you use chrome

Hello, so this wasnt a rat. if you checked the server the devs have even responded. I dont really understand how you got hacked, they have said countless time its a false positive and the only reason its getting detected is because of github.

**Comment by ThePastaGuy** [comment]
If what you say is 100% true, I still dont feel safe with a program that is giving that many flags. Not to mention the fact that my defender started freaking it after the update and set its own weird parameters not to scan itself and started crashing everytime i forced it to do so.

**Comment by Izkrul** [comment]
if you have any questions or anything you probably most likely have been answered in the discord server. understandable for you not to feel safe with the program but why would master131 & mrpond switch up on everyone and rat them after so many years put into this. that just doesnt make sense you know? if you dont trust it you can compile it yourself and look thru it or have a friend look thru it. source code is public!

discord server: https://discord.gg/mpvsaT8g

**Comment by ThePastaGuy** [comment]
I'll take my chances with the chrome extensions, tho this is pretty unfortunate bc i was a fan of this project. I hope you are totally right and my defender was just bugging out, but at this point I'm just not gonna trust modded patches for apps anymore. I'll look thru the discord.

**Comment by Izkrul** [comment]
understandable, a lot of people havent been ratted, there has been reddit posts people saying its false positives even the devs of the project said there wasnt anything "ratted". Again its your choice to run it or not. I would if you dont trust it get the older builds of it and manually download it

**Comment by robotrono** [comment]
any other alternative?

**Comment by lewill03** [comment]
> any other alternative?

Yes, [SpotX](https://github.com/SpotX-Official/SpotX) or [Spicetify](https://github.com/spicetify) (trustworthy for now...)

**Comment by ArctixTheFox** [comment]
Im no expert in C++ but the code seems fine to me.
The commits that dev pushed after the previous version (which was fine), show nothing unusual and no code was changed.

I went ahead and compiled it myself locally, uploaded it to virustotal and got a detection of [2/73](https://www.virustotal.com/gui/file/9537020e6d040d5654f667a54b16452e8b4a82edafb99ba13969311cb822a399), which is similar to the previous releases.

My theory might be a bit out there.
But is it possible the service which dev uses as CI/CD has been compromised, or at least their account?
Just taking a guess at it, seen as my locally compiled version is perfectly fine and the CI/CD one got detected by [29/73](https://www.virustotal.com/gui/file/dd78d0175c58d842c6a897e6fa9eb238b19e650bac975154674d8e4a30064ae5) engines.
